Analysis_of_rich_people.Rmd

```diff
@@ -293,7 +420,73 @@ youngest_age <- min(df[df$selfMade == TRUE, ]$age)
 cat("The age of the youngest self-made billionaire is", youngest_age, "years.")
 ```
 
-The plot indicates that young billionaires before the age of 28 most definitely inherited their wealth. The youngest self-made billionaire is 28 years old. As the age increases, there is no more pattern. The only thing worth mentioning is that the richest billionaires are mostly self-made. This probably results out of the fact of inflation and the growing gap between rich and poor.
+This indicates that young billionaires below the age of 28 most definitely inherited their wealth. The youngest self-made billionaire is 28 years old. As the age increases, there is no obvious pattern. The only thing worth mentioning is that the richest billionaires are mostly self-made. This probably results out of the fact of inflation and the growing gap between rich and poor.
+Let's zoom in on billionaires with a wealth below $25 billion:
+
+```{r}
+filtered_data <- subset(df_billionaires, finalWorth < 25000)
+
+# Scatter plot: Wealth vs. Age
+ggplot(filtered_data, aes(x = age, y = finalWorth, color = selfMade)) +
```

Comment:
Den Plot würde ich eher auf die Anzahl der milliardäre pro alter machen
Um das dann etwas besser zu sehen vielleicht auch in einer altersspanne von jeweils 5 jahren (0-4, 5-9, 10-14, ...)

Bei dem Plot hast du auch ein Warning, dass sachen rausgeschmissen werden vor dem anzeigen
`## Warning: Removed 65 rows containing missing values (`geom_point()`).`

das liegt bestimmt an der tatsache, dass NA werte drin sind.

Generell solltest du überall darauf achten die richtigen dfs zu nehmen, da das sonst auch für einen selbst zur verwirrung werden kann.

Reply:
Das gilt sowohl für diesen plot, als auch für den oberen
Die ausreißer meine ich aber nur beim oberen plot